data/data-pipeline/data_pipeline/score/score_c.py

```diff
@@ -0,0 +1,88 @@
+from data_pipeline.score.score import Score, FN, pd, logger
+
+
+class ScoreC(Score):
+    def __init__(self, df: pd.DataFrame) -> None:
+        self.BUCKET_SOCIOECONOMIC = {
```

Comment:
This is a tiny nit, but I do loved namedtuples for use cases like this where you're naming fields in a dict. It gives you a little more safety and clarity about what's happening (I was confused at first when i saw `bucket["fields"]` below).

Reply:
cool! i forgot those existed! done